rust-runtime/aws-smithy-http/src/body.rs

```diff
@@ -191,6 +196,25 @@ impl SdkBody {
             f(self)
         }
     }
+
+    /// Update this `SdkBody` with `map`. **This function MUST NOT alert the data of the body.**
```

Comment:
Should this be "MUST NOT alter"?

Reply:
Yeah. I believe it was corrected in a later PR.